client/state/jetpack-onboarding/reducer.js

```diff
@@ -27,12 +32,10 @@ export const settingsReducer = keyedReducer(
 	createReducer(
 		{},
 		{
-			[ JETPACK_ONBOARDING_SETTINGS_UPDATE ]: ( state, { settings } ) => ( {
-				...state,
-				...settings,
-			} ),
+			[ JETPACK_ONBOARDING_SETTINGS_UPDATE ]: ( state, { settings } ) =>
```

Comment:
We're now storing all of Jetpack settings, but in the onboarding settings tree? 😭

Reply:
Yes! Another total misnomer as of this PR. There's already `state.jetpack.settings` for the Redux stuff in, well, `state/jetpack/settings`, so we can't rename it to that just yet. And renaming to `jetpackSettings` for now didn't seem to make much sense if we're going to replace the stuff in `state/jetpack/settings` with it anyway, so I opted to keep all misnomers for now and address them in the de-dupe PR.